indico/modules/events/logs/client/js/actions.js

```diff
@@ -90,15 +95,15 @@ export function fetchFailed() {
 export function fetchLogEntries() {
   return async (dispatch, getStore) => {
     dispatch(fetchStarted());
-
     const {
-      logs: {filters, keyword, currentPage},
+      logs: {filters, keyword, currentPage, dataQuery},
       staticData: {fetchLogsUrl},
     } = getStore();
 
     const params = {
       page: currentPage,
       filters: [],
+      data: JSON.stringify(dataQuery)
```

Comment:
I'm very surprised that CI did not complain about a missing trailing `,` here...